### DISCUSSION TITLE
Is it possible to create a drawing tutorial video using line animation?

### DISCUSSION THREAD

**Discussion by Lukasz83POL** [keiwando/toonsquid#410] [Q&A] [answered]
Just like in these videos: https://www.youtube.com/@ligadabrincadeira/videos

**Comment by DinoAwakening** [comment]
It is actually possible to make it easy with the Path Tool, using trim properties. Anyway, basically despite this tool, as long as you can draw anything, you could literally make ANYTHING xD

**Reply by Lukasz83POL** [reply]
Yes, I know I can draw in Toonsquid, but is it possible to animate the drawing creation process? Or what can be used to do something like this but not Videoscribe.

**Reply by DinoAwakening** [reply, marked as answer]
I think you might lost my explanation(? Or maybe my explanation was very poor(? The thing is; “Path Tool” is just the tool you’re looking for to make this kind of videos since you basically can animate simulated traces using “Trim” function in properties in combination with Tweening features.

**Reply by Lukasz83POL** [reply]
Thanks for the answer! I actually didn't fully understand it the first time, but I'll definitely check it out now. Best Regards!
